src/file.cr

```diff
@@ -16,7 +16,7 @@ class File < IO::FileDescriptor
   {% end %}
 
   # :nodoc:
-  DEFAULT_CREATE_MODE = LibC::S_IRUSR | LibC::S_IWUSR | LibC::S_IRGRP | LibC::S_IROTH
+  DEFAULT_CREATE_PERMISSIONS = 0o644
```

Comment:
Maybe it is better to use `File::Permissions.flags(OwnerRead, OwnerWrite, GroupRead, OtherRead)` here?

Reply:
I don't think so, the shorter octal syntax is far easier to read. "everyone" knows unix octal permission syntax.

Reply:
Then why bother with a `File::Permissions` enum?